Vagrantfile

```diff
@@ -47,6 +50,26 @@ Vagrant.configure(2) do |config|
     do
         echo unset $LC  >> /home/vagrant/.bashrc
     done
+
+    # Install VirtualBox guest additions
+    cd /tmp
+    VBOX_VERSION=$1
+    VBOX_ADDITIONS_URL="http://download.virtualbox.org/virtualbox/VER/VBoxGuestAdditions_VER.iso"
+    sudo wget -q -c ${VBOX_ADDITIONS_URL//VER/$VBOX_VERSION} -O VBoxGuestAdditions_$VBOX_VERSION.iso
```

Comment:
Why `wget -q`?  We hide the errors when it fails, which forces you to rerun `vagrant up` without it.  Also, why `-c`?  We can't be continuing anything because this iso can't be in `/tmp` at this point.